### PR TITLE
New cookie feature

### PR DESCRIPTION
**Pull request by paramt**
```
[] =  required argument
<> = optional argument
```
```
cookie give [user] <amount>
````
```
cookie balance <user>
````

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
 from club.cakebot import EmbedUtil, ServerUtil, TextCommandsUtil, Bootstrap
 from lcbools import false
 from bs4 import BeautifulSoup as HTML
+from cookies import Cookie
```
Comment (RDIL) [inline main.py]: wha

Comment (paramt) [reply main.py]: I was thinking of maintaining [a separate module](https://github.com/cakebotpro/cookies/blob/master/Cookie/__init__.py) for the cookie feature. If you think it'd be easier we can just do everything in one place

Comment (RDIL) [reply main.py]: no its fine

**Comment by RDIL** [comment]
Made #36 as a todo list for this

**Pull request merged**